src/ast/irbuilderbpf.h

```diff
@@ -185,6 +185,9 @@ class IRBuilderBPF : public IRBuilder<>
                     size_t size,
                     const location *loc = nullptr);
   void CreateAtomicIncCounter(int mapfd, uint32_t idx);
+  void CreateDebugOutput(std::string fmt_str,
```

Comment:
```suggestion
  void CreateDebugOutput(const std::string &fmt_str,
```
Or use `std::string_view`, but that would probably require a little more work on the caller's side